tests/by-util/test_dmesg.rs

```diff
@@ -30,3 +30,55 @@ fn test_kmsg_json() {
         .no_stderr()
         .stdout_is_templated_fixture("test_kmsg_json.expected", &[("\r\n", "\n")]);
 }
+
+#[test]
+fn test_kmsg_time_format_delta() {
+    test_kmsg_time_format("delta");
+}
+
+#[test]
+fn test_kmsg_time_format_reltime() {
+    test_kmsg_time_format("reltime");
+}
+
+#[test]
+fn test_kmsg_time_format_ctime() {
+    test_kmsg_time_format("ctime");
+}
+
+#[test]
+fn test_kmsg_time_format_notime() {
+    test_kmsg_time_format("notime");
+}
+
+#[test]
+fn test_kmsg_time_format_iso() {
+    test_kmsg_time_format("iso");
+}
+
+#[test]
+fn test_kmsg_time_format_raw() {
+    test_kmsg_time_format("raw");
+}
```

Comment:
What's the reason for having a test function for each format? Why not loop over the formats in `test_kmsg_time_format`?

Reply:
The issue I had with looping over the formats in a single test function is that once a format specified earlier in the loop fails, the other formats weren't tested. This was annoying for me especially when developing the features, since I wrote the test for all formats first, but then I implemented each feature separately in no particular order. It might not be necessary anymore now that it's all implemented, but it is still better, in my opinion, since it's more verbose. I'll go with your direction.

Initially, I want to write it like the `[TestCase]` attribute from `NUnit` (https://docs.nunit.org/articles/nunit/writing-tests/attributes/testcase.html), however upon researching, there is no built-in equivalent in Rust.

Reply:
One other option, we can implement a macro to get rid of the repeated code while still having separate test for each `time-format` value.

Reply:
I think a macro is overkill in this case, it adds complexity without much benefit.

Reply:
Got it, so should I just change it to a loop then?

Reply:
No, you can leave it as it is.